test/parallel/test-cluster-inspector-debug-brk.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+// This test ensures that flag --inspect-brk and --debug-brk works properly
+
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+const script = common.fixturesDir + '/empty.js';
+
+function fail() {
+  assert(0); // `node --debug-brk script.js` should not quit
+}
+
+if (cluster.isMaster) {
+
+  function fork(execArgv) {
+    cluster.setupMaster({execArgv});
+    let worker = cluster.fork();
+
+    worker.on('exit', fail);
+
+    // give node time to start up the debugger
+    setTimeout(() => {
+      worker.removeListener('exit', fail);
+      worker.kill();
+    }, 2000);
```

Comment:
this looks like a recipe for a flaky test, I'd suggest waiting until the debug/inspect TCP port is open/can be connected to